src/plugins/dashboard/public/dashboard_app/no_data/dashboard_app_no_data.tsx

```diff
@@ -54,7 +125,7 @@ export const DashboardAppNoDataPage = ({
 
   return (
     <AnalyticsNoDataPageKibanaProvider {...analyticsServices}>
-      <AnalyticsNoDataPage onDataViewCreated={onDataViewCreated} />
+      <AnalyticsNoDataPage onDataViewCreated={onDataViewCreated} onTryESQL={onTryESQL} />
```

Comment:
AnalyticsNoDataPage does not show any loading indication when clicked. This means that it can be click multiple times. In this case, abortController from previous click is not canceled. There are 2 fixes, 1) disable button on first click. 2) cancel previous request abortController when `onTryESQL` is called again

Reply:
Done https://github.com/elastic/kibana/pull/196658/commits/f806040487bcc820e8680b3e4c4af369ae5fe0c0